packages/desktop-electron/index.js

```diff
@@ -76,11 +76,11 @@ if (isDev) {
 }
 
 function createBackgroundProcess(socketName) {
-  serverProcess = fork(__dirname + '/server.js', [
-    '--subprocess',
-    app.getVersion(),
-    socketName,
-  ]);
+  serverProcess = fork(
+    __dirname + '/server.js',
+    ['--subprocess', app.getVersion(), socketName],
+    isDev ? { execArgv: ['--inspect'] } : undefined,
```

Comment:
Allows opening `chrome://inspect/#devices` to inspect the backend server remotely